crates/analytics/docs/clickhouse/scripts/payouts.sql

```diff
@@ -74,21 +92,28 @@ CREATE MATERIALIZED VIEW kafka_parse_payout TO payout (
     `return_url` Nullable(String),
     `entity_type` LowCardinality(String),
     `metadata` Nullable(String),
-    `created_at` DateTime64(3),
-    `last_modified_at` DateTime64(3),
+    `created_at` DateTime CODEC(T64, LZ4),
```

Comment:
we don't need compression codec over here. the mv is only a connector between queue and destination tables